### PR TITLE
Update style_guide.md

### PR DESCRIPTION
**Pull request by floradanna**
make bibliography as header

### CODE REVIEW

File: pages/contribute/style_guide.md [hunk 1]
```diff
@@ -42,17 +42,18 @@ In general, we follow the European Commission's [Web Writing Style Guide](https:
...
   * **That/which:** use "that" when you are defining something and "which" when you are adding extra information about it e.g.:
     * "The cat that was on the table suddenly got up" is telling us which cat it was. It is important to the meaning of the sentence because you are not talking about any cat, just the cat on the table.
     * "The cat, which was sitting on the table, suddenly got up" is giving us extra information about the cat. The information is not necessary to understand the sentence. You can remove the clause and the sentence will still be clear. Clauses starting with "which" usually begin with a comma.
   * **Titles (the "title" in the front matter of pages):** only the first word, proper nouns and acronyms are capitalised.
   * **Tool assembly:** there are multiple tools in **one** assembly. The plural is "tool assemblies".
   * **Training:** training is an uncountable noun and cannot have a plural. You can write "training courses" and "training materials" but not "trainings".
 
+## Bibliography
+ * add your citations as bibtex to the `_bibliography/references.bib` file
```
Comment (martin-nc) [inline pages/contribute/style_guide.md:53]: I think each bullet point should start with a capital here (e.g. "Add your citations...") and end in a full stop.

File: pages/contribute/style_guide.md [hunk 2]
```diff
@@ -42,17 +42,18 @@ In general, we follow the European Commission's [Web Writing Style Guide](https:
...
+ * add `{% cite reference_key %}` to the text where you are citing the reference
+ * add  `## Bibliography` `{% bibliography --cited %}` to show a bibliography section with cited refrences on a page.
```
Comment (martin-nc) [inline pages/contribute/style_guide.md:55]: "References" is spelled wrong, and perhaps rephrase to: "Add `## Bibliography` `{% bibliography --cited %}` to show a bibliography section containing the references in the page."

**Review by martin-nc — COMMENTED** [review]
@floradanna I've just added a couple of comments - what do you think?

File: pages/contribute/style_guide.md (hunk 1 above)
Comment (floradanna) [reply pages/contribute/style_guide.md:53]: ok for me

File: pages/contribute/style_guide.md (hunk 2 above)
Comment (floradanna) [reply pages/contribute/style_guide.md:55]: ok for me. @bedroesb is it possible? what do you think?

Comment (bedroesb) [reply pages/contribute/style_guide.md:55]: Sure! If I understand correctly this is just changing the last sentence and not the snippets used by the script.

**Pull request merged**